partiql-app/partiql-cli/shell.sh

```diff
@@ -17,4 +17,4 @@ cli_path=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
 
 cd "$cli_path"
 ../../gradlew :partiql-app:partiql-cli:install
-../partiql-cli/build/install/partiql-cli/bin/partiql
+../partiql-cli/build/install/partiql-cli/bin/partiql "$@"
```

Comment:
Depending on how well argument expansion/passthrough to the gradle generated script works here (I definitely did not thoroughly test this at the time) we could rename this script to `shell.sh` as it is now more than just a way to start the REPL

Reply:
> ... rename this script to `shell.sh`

It's already `shell.sh`. Do you mean something like `partiql`? I'd be interested in creating a root-level directory called `bin` or `partiql-bin`.

Then, from the project root, we can run `./bin/partiql`. And, if users want, they can add the bin to their PATH, so they can just run `partiql`. On top of that, I haven't looked too much into it -- but I'd like the current script to not have to re-install and re-assemble on every invocation. I know we can set the Gradle task's inputs/outputs to avoid frequent assembly, but I haven't made any attempts yet.

Reply:
I definitely meant `partiql.sh` haha and just made a mistake

Reply:
Robert from two days ago had too much egg nog

Reply:
Renamed!